bc/subscription/tasks.py

```diff
@@ -35,9 +36,21 @@ def process_filing_webhook_event(fwe_pk) -> FilingWebhookEvent:
     filing_webhook_event.subscription = subscription
     filing_webhook_event.save()
 
-    message = MASTODON_POST_TEMPLATE.format(
+    if filing_webhook_event.short_description:
+        description = filing_webhook_event.short_description
+    else:
+        docket_length = len(subscription.docket_name)
+        available_space = 500 - len(POST_TEMPLATE) - docket_length
+        if len(filing_webhook_event.long_description) > available_space:
+            description = trunc(
+                filing_webhook_event.long_description, available_space, "...👇"
```

Comment:
There's an ellipsis unicode character that looks better and only takes one byte from the length of our posts: `…`. But also, we shouldn't do this until we have the photo part ready.

Reply:
got it!